must-gather/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM quay.io/openshift/origin-must-gather:4.2.0
+FROM quay.io/openshift/origin-must-gather:latest
```

Comment:
Is there not a fixed tag that uses this yet? Also, is this being changed in OCP 4.2 as well?

Reply:
I was thinking of using the latest image with master branch. As the downstream build is happening from the 4.2 branch we can have the latest fix in the master I guess.

Reply:
Ah! Okay. Well, as long as it doesn't hurt for now that's okay.